components/inputs/selects/DropdownMultiSelect.tsx

```diff
@@ -120,70 +126,78 @@ export const DropdownMultiSelect: React.FC<DropdownMultiSelectProps> = ({
                 {...getToggleButtonProps()}
               >
                 <div className={styles.container}>
-                  <div
-                    className={styles.selectedMeasure}
-                    ref={contentRef as React.LegacyRef<HTMLDivElement>}
-                  >
-                    {selectedItems.map(sel => {
-                      return (
-                        <div key={sel.label} className={styles.selectedWrapper}>
-                          {sel.component}
+                  {simple && <div className={styles.label}>{label}</div>}
```

Comment:
I would suggest to move this part to method and render it like
```
if (simple) {
  return <div className={styles.label}>{label}</div>;
}

return *everything else here*
```

In general I would suggest to break rendering of the component to multiple methods as JSX is very messy and it's really hard to understand what's going on